packages/tlshelpers/PKGBUILD

```
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=tlshelpers
+pkgver=21.6e422be
+pkgrel=1
+pkgdesc='A collection of shell scripts that help handling X.509 certificate and TLS issues.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-recon')
+url='https://github.com/hannob/tlshelpers'
+license=('custom:UNLICENSE')
```

```suggestion
license=('Unlicense')
```

cf. `/usr/share/licenses/common/Unlicense/`